src/consensus/pbft/libbyz/Replica.h

```diff
@@ -427,10 +441,24 @@ class Replica : public Node, public IMessageReceiveBase
   Rep_info replies;
 #endif
 
-  // used to register a callback for a client proxy to collect replies sent to
-  // this replica
+  ByzInfo playback_byz_info;
```

Comment:
these are added as playback state, mirroring `execute_tentative()` when it has multiple requests to execute for one pre-prepare